rbac/rbac.go

```diff
@@ -49,7 +49,7 @@ func WithContext(ctx context.Context, scope *Scope) context.Context {
 func ExtractScopeFromHeader(r *http.Request) *Scope {
 	groupStr := r.Header.Get(ScopeHeader)
 	tagsStr := r.Header.Get(ScopeReleaseTagsHeader)
-	if len(groupStr) > 0 {
+	if len(groupStr) || len(tagsStr) > 0 {
```

Comment:
just to double check: are we sure that for else to this if we are returning nil?

Reply:
sorry, but I don't understand the comment; can you rephrase?

Reply:
that is something we were already doing (btw, I fixed the condition) and is something we don't want to change

Reply:
copy that

Reply:
I noticed the condition fix. and I was copying this code to the comment -- still failed to notice the new syntax ;-)